typings/eslint-plugin-vue/util-types/ast/es-ast.ts

```diff
@@ -347,6 +347,7 @@ export interface PrivateIdentifier extends HasParentNode {
 export interface Literal extends HasParentNode {
   type: 'Literal'
   value: string | boolean | null | number | RegExp | BigInt
+  raw: string
```

Comment:
`console.log(node)`, the following object can be displayed.
The fixer use the `raw` property to determine the single/doublequotes.

```sh
<ref *2> Node {
  type: 'Literal',
  start: 38,
  end: 51,
  loc: SourceLocation {
    start: { line: 3, column: 10 },
    end: { line: 3, column: 23 }
  },
  range: [ 38, 51 ],
  value: 'MyComponent',
  raw: "'MyComponent'",
  parent: <ref *1> Node {
    type: 'Property',
    start: 32,
    end: 51,
    loc: SourceLocation { start: [Object], end: [Object] },
    range: [ 32, 51 ],
    method: false,
    shorthand: false,
    computed: false,
    key: Node {
      type: 'Identifier',
      start: 32,
      end: 36,
      loc: [SourceLocation],
      range: [Array],
      name: 'name',
      parent: [Circular *1]
    },
    value: [Circular *2],
    kind: 'init',
    parent: Node {
      type: 'ObjectExpression',
      start: 26,
      end: 110,
      loc: [SourceLocation],
      range: [Array],
      properties: [Array],
      parent: [Node]
    }
  }
}
```

Reply:
I found `Literal`'s `raw` property is defined [here](https://github.com/vuejs/vue-eslint-parser/blob/c476e7c25ab161b336339448765085e6219d0e04/src/ast/nodes.ts#L392).